index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Inter">
+    <title>Portfolio</title>
+</head>
+<body>
+    <header>
+    <div class="nav-container">
+
+        <nav class="nav-bar">
+            <div class="nav-toggle"><img src="images/Enabled.png"></div>
+            <div class="nav-item" id="box1"><a href="#">Portfolio</a></div>
+            <div class="nav-item" id="box2"><a href="#">About</a></div>
+            <div class="nav-item" id="box3" ><a href="#"> Contact</a></div>
+        </nav>
+    </div>
+    </header>
+    <div class="main-container">
```

Comment:
great work on wrapping the main work in a div but it's better to use a main tag as container as you might have many div tag inside the same section this will make you code clear organized and clean